### ISSUE TITLE
`RefCounted.getReferenceCount()`: `expected signed integer type, found 'u64'`

### ISSUE THREAD

**Issue by Des-Nerger** [thimenesup/GodotZigBindings#15]
```diff
--- a/test_example/src/test_node2d.zig
+++ b/test_example/src/test_node2d.zig
@@ -61,3 +61,4 @@ pub const TestNode2D = struct {
     pub fn _ready(self: *Self) void {
-        _ = self;
+        if (self.object_property) |texture2d|
+            std.debug.print("_ready texture2d.ref_count:{}", .{texture2d.base.base.base.getReferenceCount()});
```
```
src/godot.zig:174:30: error: expected signed integer type, found 'u64'
            return @truncate(@as(u64, @bitCast(ret)));
                             ^~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    getReferenceCount: src/gen/classes/ref_counted.zig:52:34
    _ready: src/test_node2d.zig:63:82
```

**Comment by Des-Nerger** [comment]
Also, a related question: how do we free a `RefCounted` object when `.unreference()` returns `true`? In C++ they do this in the destructor of `Ref`:
```c++
// godot-cpp/classes/ref.hpp
~Ref() {
	unref();
}
void unref() {
	if (reference && reference->unreference()) {
		memdelete(reference);
	}
	reference = nullptr;
}
```
But in GodotZigBindings I haven't found neither `_memdelete()` nor `free()` that I could call on a `RefCounted` descendant. Did I miss something?
~~**UPD**: Maybe we can free them like this, for now:~~
~~`var free = StringName.initUtf8("free");`~~
~~`defer free.deinit();`~~
~~`_ = texture2d.base.base.base.base.call(&free, .{});`~~
**UPD2**: Nah, I was wrong, the `.call(&free, .{})` trick didn't work out: Godot gave this: `ERROR: Can't 'free' a reference`.

**Comment by thimenesup** [comment]
Main issue fixed by c020d1c.
